src/algebra/group_with_zero.lean

```diff
@@ -182,6 +182,9 @@ units.ext rfl
   units.mk0 a ha = units.mk0 b hb ↔ a = b :=
 ⟨λ h, by injection h, λ h, units.ext h⟩
 
+@[simp] lemma exists_iff_ne_zero (x : G₀) : (∃ u : units G₀, ↑u = x) ↔ x ≠ 0 :=
```

Comment:
Please move `is_unit.mk0` and `is_unit_iff_ne_zero` from `algebra.associated` instead.

Reply:
Done